docs/sources/release-notes/v2.2.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Grafana Mimir version 2.2 release notes"
+menuTitle: "V2.2 release notes"
+description: "Release notes for Grafana Mimir version 2.2"
+weight: 300
+---
+
+# Grafana Mimir version 2.2 release notes
+
+Grafana Labs is excited to announce version 2.2 of Grafana Mimir, the most scalable, most performant open source time series database in the world.
+
+Highlights include the top features, enhancements, and bugfixes in this release. If you are upgrading from Grafana Mimir 2.1, there is migration-related information as well.
+For the complete list of changes, see the [Changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
```

Comment:
I would call out the Helm chart too. It's true it has its own release cycle and changelog, but we should call it out as a massive improvement since Mimir 2.1.0. @krajorama and @Logiraptor could help drafting the notes.

Reply:
Added placeholder for now:

> - **Helm Chart update**: TBD